Add xunit sample to Scorecard Docs (#5762)
operator-framework/operator-sdk: website/content/en/docs/testing-operators/scorecard/_index.md

}
```

### XML format

> See below an example of XMl format produced by a scorecard test. Please note that the XMl is formatted to be compatible with the XUnit schema for post-processing the results.

Reviewer: ```suggestion
See the example below for the results of a scorecard test fromatted in XML. Please note that the XML is formatted to be compatible with the XUnit schema for post-processing the results.
```
A couple of questions:
Should "Please note..." be a callout box?
Does the scorecard tool format the output for the XUnit schema, or did you do that? If scorecard does it, maybe a slight tweak would make it clearer: "The scorecard tool formats the XML output for XUnit schema compatability. This format makes it easier the test results.
